src/com/goide/inspections/GoStructInitializationInspection.java

```diff
@@ -36,8 +36,11 @@ public void visitCompositeLit(@NotNull GoCompositeLit o) {
         if (PsiTreeUtil.getParentOfType(o, GoReturnStatement.class, GoShortVarDeclaration.class, GoAssignmentStatement.class) == null) return;
 
         GoType type = o.getType();
-        if (!(type instanceof GoStructType) &&
-            !(GoPsiImplUtil.findBaseTypeFromRef(o.getTypeReferenceExpression()) instanceof GoSpecType)) {
+        GoType typeFromTypeRef = GoPsiImplUtil.findTypeFromTypeRef(o.getTypeReferenceExpression());
+        if (!(type instanceof GoStructType ||
```

Comment:
I think the find method will bubble up too much and resolve to the type ` string ` instead of the type ` Item `. Maybe the change should be done in that function https://teamcity.jetbrains.com/viewLog.html?buildId=693727&buildTypeId=IntellijIdeaPlugins_Go_Test